client/src/test/suite/extension.test.ts

```diff
@@ -306,4 +313,34 @@ suite('Extension Test Suite', function (this: Suite) {
     populateXsdSchemaFiles(TEST_OUT_DIR)
     assert(!fs.existsSync(testXsdPath))
   })
+  test('push with no conflict', async () => {
+    stubRepo.commit = sinon.stub().throws()
+    stubRepo.pull = sinon.stub().throws()
+    stubRepo.push = sinon.stub().throws()
+    await pushContent.pushContent()
+  })
+  test('push with merge conflict', async () => {
+    try {
+      stubRepo.commit = sinon.stub().throws(GitErrorCodes.Conflict)
```

Comment:
the `commit` function can never throw an error formed this way. Also, `pushContent` should **not** throw in the case of a conflict, so the assertion in this test doesn't seem correct